resources/lang/en/billing.php

```diff
@@ -68,4 +68,6 @@
     'saveAsAutoPayMethodBank' => 'Automatically charge this bank account for all future invoices',
     'bankAccountAdded' => 'Bank account added successfully.',
     'cvc' => 'CVC',
+    'addNewGoCardless' => 'Add New GoCardless',
```

Comment:
We shouldn't expose this to the customer as 'GoCardless', they won't necessarily know what it means. Add bank account or direct debit would be better.

Reply:
Removed the `GoCardless` reference. It will now show up as bank account for both bank account and GoCardless setups.